scripts/npm/colors/index.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var path = require( 'path' );
+var fs = require( 'fs' );
+var moment = require( 'moment' );
+var parseCSV = require( 'csv' ).parse;
+var transformCSV = require( 'csv' ).transform;
+var root = path.join( __dirname, '..', '..', '..' );
+var templateSource = path.join( root, 'scripts', 'npm', 'colors', 'template.txt' );
+var colorsSource = path.join( root, '_data', 'cfpb-brand-colors.csv' );
+var colorsLess = path.join( root, 'src', 'static', 'css', 'brand-palette.less' );
+var family;
+
+var transformer = transformCSV( function( data ) {
+  var block = '';
+  if ( data[0] === 'family' ) {
+    return fs.readFileSync( templateSource )
+             .toString()
+             .replace( /\[date\]/g, moment().format( 'MMMM Do YYYY, h:mm:ss a' ) );
```

Comment:
There's no really way to accidentally run the script. You have to purposefully do it and we can catch that change in a PR. This is definitely the easiest way, otherwise you have to update the template manually and it's yet another step to remember. I'm trying to remove steps.